app/kuma-dp/pkg/dataplane/metrics/metrics_format_mapper.go

```diff
@@ -183,18 +183,20 @@ func toOpenTelemetryQuantile(prometheusQuantiles []*io_prometheus_client.Quantil
 	return otelQuantiles
 }
 
-func extraAttributesFrom(mesh string, dataplane string, service string, extraLabels map[string]string) []attribute.KeyValue {
+func extraAttributesFrom(mesh string, dataplane string, service string, attributes map[string]string, unifiedResourceNamingEnabled bool) []attribute.KeyValue {
 	var extraAttributes []attribute.KeyValue
-	if mesh != "" {
```

Comment:
I don't get this, was this just passed for backward compatibility or something? Because it looks to me that these extraLabels are already passed in in the dynconfig no?

Reply:
If that's the case it might actually to do this kind of expansion earlier and stop having to pass through mesh, dp, service, unifiedResourceNamingEnabled in 2 functions no?

Reply:
it was used to add some stuff we were adding in old metrics, for example tags from dataplane, we can now use it to add workload, and potentially some extra labels which could be configured from policy in the future